lib/util/jsonUtils.ts

```diff
@@ -0,0 +1,121 @@
+import { Suppression, SuppressionItem } from '@azure/openapi-markdown'
```

Comment:
can you point in these new files what was changed beyond splitting the old one please ?